packages/phone-number-privacy/combiner/src/config.ts

```diff
@@ -165,6 +175,7 @@ if (DEV_MODE) {
         currentVersion: Number(functionConfig.domains_keys.current_version),
         versions: functionConfig.domains_keys.versions,
       },
+      timeoutMs: Number(process.env.TIMEOUT_MS ?? FULL_NODE_TIMEOUT_IN_MS),
```

Comment:
Same as above: 
I know I said to pull this from an env variable, but what I meant was actually to pull it from the config. That's my bad for not being clear. Could you please change this to Number(functionConfig.blockchain.timeout_ms ?? FULL_NODE_TIMEOUT_IN_MS)

Note that capital letters aren't allowed in the functionConfig field names